test/data/build-rpms-and-repos.sh

```diff
@@ -32,6 +32,7 @@ function build_rpms() {
 
     # Note: _build_name_fmt requires escaped %% for use in headerSprintf()
     rpmbuild -ba \
+        --dbpath "${RPMBUILD_TOPDIR}" \
```

Comment:
This means it cannot read dependencies installed on the system for building packages.

Reply:
And this is a problem because we don't use `--no-deps`.

Reply:
Is there a possibility to add build-time dependencies to test packages? They don't have any and testcases will pass right now.

Reply:
Maybe? I'm not sure.